Crowdsignal's OAuth2 Login Page Style Update

ice9js opened the pull request:
## Changes proposed in this Pull Request

This is an attempt at restyling the login page for Crowdsignal OAuth2 client according to pabtAt-8H-p2.
Because of changes to `client/layout/masterbar/oauth-client.scss` this PR relies on #30709 to be merged first.

![logincrowdsignal](https://user-images.githubusercontent.com/8056203/53045911-19d70380-348f-11e9-897b-42a9c9d9cd0f.png)

## Testing instructions

The changes are purely cosmetic so no functionality should be affected.
Go to https://app.crowdsignal.com and click *Sign in with WordPress.com* to access the current login page and update the domain to match your testing environment.

- Verify you're able to log in using your WordPress.com credentials.
- Verify an appropriate error is displayed when you enter a wrong email or password.
- Verify you can log in using a Google account.

pgk commented on client/login/wp-login/login-links.jsx:
```diff
@@ -237,6 +239,17 @@ export class LoginLinks extends React.Component {
 
+		if ( config.isEnabled( 'signup/wpcc' ) && isCrowdsignalOAuth2Client( oauth2Client ) ) {
+			const oauth2Flow = 'crowdsignal';
+			const redirectTo = get( currentQuery, 'redirect_to', '');
```
@ice9js Wondering if this needs to be made more specific, i.e. not accept [open redirects](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.md) or if this is handled elsewhere in the Calypso code

ice9js replied:
It's handled by a check that validates `redirect_to` against the given `oauth2_client_id` when the page is loaded so it should be fine.

pgk replied:
Got it! You mean the whole url or the id?

ice9js replied:
Specifically, it checks if the url from `redirect_to` matches the domain that's specified for the current OAuth2 client, if that's what you're asking.

pgk replied:
Thanks, got me covered! :) 💯

ice9js commented:
@digitalwaveride I addressed all the issues you mentioned except the font - I prefer to handle that separately for both login and signup screens. Please have another look when you have a minute to spare :)

digitalwaveride commented:
Looks great! Thanks Kuba!
Just one tiny tiny thing I spotted: The "or" in the middle of the screen has a font-size of 25px
the `class="login__form-subheader"` has font-size 22px.

Would be great to change both to 22px.

ice9js commented:
> Would be great to change both to 22px.

Done :)
